<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventas</class>
 <widget class="QMainWindow" name="Ventas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Pedidos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qlineargradient(spread:reflect, x1:0.502127, y1:0, x2:0.508, y2:1, stop:0.290837 rgba(16, 250, 255, 255), stop:1 rgba(16, 60, 255, 255));

border-radius:20px;

border:1px solid #00007f;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneCodigo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnePizza">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pizza</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneCantidad">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGrabar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 55, 55);
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblVentas">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>811</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 139, 68);</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pizza</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Costo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Monto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de Pedido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de Entrega</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Factura</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lneFechadePedido">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>FechadePedido</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Costo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneCosto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>FechadeEntrega</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneFechadeEntrega">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneFactura">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>GenerarFactura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>811</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
border:None;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../imagenes del qtdesigner/pizzas-mobile_v3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>lneCodigo</zorder>
   <zorder>lnePizza</zorder>
   <zorder>label_2</zorder>
   <zorder>lneCantidad</zorder>
   <zorder>label_4</zorder>
   <zorder>btnGrabar</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>lneFechadePedido</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lneCosto</zorder>
   <zorder>label_7</zorder>
   <zorder>lneFechadeEntrega</zorder>
   <zorder>tblVentas</zorder>
   <zorder>lneFactura</zorder>
   <zorder>label_8</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lneCodigo</tabstop>
  <tabstop>lnePizza</tabstop>
  <tabstop>lneFechadePedido</tabstop>
  <tabstop>lneCantidad</tabstop>
  <tabstop>btnGrabar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>tblVentas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
